### DISCUSSION TITLE
使用FTP存储无法正常挂载，“Request failed with status code 500”

### DISCUSSION THREAD

**Discussion by goodrabbit** [AlistGo/alist#1738] [General]
在Alist v3 - rc1 版本中，在管理后台使用FTP存储无法实现正常挂载。虽然定义上设置没错，也能确认定义成功，但在Web界面上却有如下错误：
“Request failed with status code 500”

另，使用阿里云盘的其他我的几个帐户定义存储，目前还是正常的。不知这个FTP为何有错？
![image](https://user-images.githubusercontent.com/5088561/191457192-1c37930c-fa76-4b46-8768-ec2601811d39.png)

**Comment by chqcxd** [comment]
求助，我也是这个情况，raidrive挂载ftp没有问题的。

**Comment by goodrabbit** [comment]
谢谢@Xhofe老大，已经确认你在3.0.1的发布版中解决了，测试有效，太好了！！
